redhat-support-lib/bin/README.md

```diff
@@ -0,0 +1,4 @@
+redhat-support-lib-java
+=======================
+
+Java library for interracting with Red Hat's Strata API
```

Comment:
You are correct. I must have left it in there by mistake. Removing this should fix a large amount of the issues noted.